module/icp/algs/blake3/blake3.c

```diff
@@ -25,6 +25,7 @@
  * Copyright (c) 2021-2022 Tino Reichardt <milky-zfs@mcmilk.de>
  */
 
+#include <sys/simd.h>
```

Comment:
When this `hack` sith `<sys/simd.h>` is needed first, we should maybe comment this also.
This would be needed by the other places as well.

Reply:
zfs_context.h includes vnode.h and we need elf_common.h to be processed before vnode.h otherwise the hack does not work

Reply:
Again, for the hack in vnode.h to work, simd.h needs to be always included before vnode.h